src/echo.js

```diff
@@ -23,12 +27,23 @@ function _echo(opts, messages) {
   // allow strings starting with '-', see issue #20
   messages = [].slice.call(arguments, opts ? 0 : 1);
 
-  if (messages[0] === '-e') {
-    // ignore -e
+  var option = ['-e', '-n', '-ne', '-en'].indexOf(messages[0]);
+  var output;
+
+  if (option >= 0) {
```

Comment:
I think we can use `common.parseOptions` to parse the first argument. If we catch the `option not recognized` error, then just print that string.